QuasarHelios2015/src/org/team1540/quasarhelios/ControlInterface.java

```diff
@@ -79,7 +79,7 @@ private static void setupRollers() {
         Igneous.joystick2.getAxisSource(2).send(Rollers.leftRollerOverride);
         Igneous.joystick2.getAxisSource(6).send(Rollers.rightRollerOverride);
 
-        FloatInput cutoffAuto = mainTuning.getFloat("autoloader-button-threshold", 0.5f);
+        FloatInput cutoffAuto = mainTuning.getFloat("Loader Button Threshold +M", 0.5f);
```

Comment:
Trigger Threshold might actually be a better name for this, since it's both AutoLoader and AutoEjector